### ISSUE TITLE
Mandating colouration and typography defined by the system to improve accessibility causes disablement of all iconography.

### ISSUE THREAD

**Issue by BEEDELLROKEJULIANLOCKHART** [vector-im/element-web#19820]
### Steps to reproduce:
Configure “`this`” to be “`this`”, and “`this`” to be “`this`” via “`firefox --new-tab about:config`”.

### Outcome:
No iconography is presented.

#### What did you expect?
Iconography should be presented.

#### What happened instead?
What ![this](http://user-images.githubusercontent.com/42837531/142706207-971e63e8-c2ad-4489-b64f-9b29366aeadb.png) depicts.

### Operating system
[GNU Software](https://www.gnu.org/software/gsrc/package-list.html) and the kernel of [Linux](http://wikipedia.org/wiki/Linux).

### Browser information
Mozilla Firefox Flatpak 95.0b9 (64-bit).

### URL for webapp:
“http://develop.element.io”.

### Application version
“Element version: b2e8f212e410-react-7f6f98443865-js-b2d83c1f80b5 Olm version: 3.2.3”.

### Homeserver:
“matrix.org”.

### Will you send logs?
Yes.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
![This does not appear to remain.](https://user-images.githubusercontent.com/42837531/182484477-6f433afa-be84-4b4a-91dc-67c7c09936a0.png)

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
![This does not appear to remain.](https://user-images.githubusercontent.com/42837531/182484477-6f433afa-be84-4b4a-91dc-67c7c09936a0.png)

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
![This](https://user-images.githubusercontent.com/42837531/182485258-c07e5c3e-7de2-4fd0-af0b-3fc76f2ef688.png) demonstrates that I was incorrect because I somehow did not realize that I had not enabled the relevant options. Tiredness, eh.
